_episodes/05-loop.md

```diff
@@ -362,12 +362,31 @@ from whatever file is being processed
 > ```
 > {: .output}
 > Try removing the quotes around `$filename` in the loop above to see the effect of the quote
-> marks on whitespace:
+> marks on whitespace. Note that we get a result from the loop command for unicorn.dat if you ran this code in the creatures directory:
 > ```
 > head: cannot open ‘red’ for reading: No such file or directory
 > head: cannot open ‘dragon.dat’ for reading: No such file or directory
 > head: cannot open ‘purple’ for reading: No such file or directory
-> head: cannot open ‘unicorn.dat’ for reading: No such file or directory
+> CGGTACCGAA
+> AAGGGTCGCG
+> CAAGTGTTCC
+> CGGGACAATA
+> GTTCTGCTAA
+> GATAAGTATG
+> TGCCGACTTA
+> CCCGACCGTC
+> TAGGTTATAA
+> GGCACAACCG
+> CTTCACTGTA
+> GAGGTGTACA
+> AGGATCCGTT
+> GCGCGGGCGG
+> CAGTCTATGT
+> TTTTCGACAC
+> TGGACTGCTT
+> CCCTTTGAGG
+> GTGGATTTTT
+> CGTAACGGGT
```

Comment:
Unfortunately this takes up a lot of vertical space, just due to the command used in the example (and this callout box was already pretty big to begin with).

What do you think about changing the tail command to something like `tail -n 3`? (You might also want to change the code outside the callout box to match, but I don't think it's totally necessary.)

Reply:
Thanks for the suggestions - I made all of these changes!